clients/src/main/resources/common/message/ShareGroupHeartbeatRequest.json

```diff
@@ -34,13 +34,6 @@
     { "name": "RackId", "type": "string", "versions": "0+",  "nullableVersions": "0+", "default": "null",
       "about": "null if not provided or if it didn't change since the last heartbeat; the rack ID of consumer otherwise." },
     { "name": "SubscribedTopicNames", "type": "[]string", "versions": "0+", "nullableVersions": "0+", "default": "null",
-      "about": "null if it didn't change since the last heartbeat; the subscribed topic names otherwise." },
-    { "name": "TopicPartitions", "type": "[]TopicPartitions", "versions": "0+", "nullableVersions": "0+", "default": "null",
```

Comment:
I agree that we don't use `TopicPartitions` currently in share group heartbeat, also consumer group uses to fetch current assigned `TopicPartitions` hence will not this be needed when we move away from simple assignor i.e. we assigned shared partitions differenty and in heartbeat required the current owned partition by client?

Reply:
We'd do a v2 RPC once we had worked out what we actually wanted to achieve.